datafusion/core/tests/row.rs

```diff
@@ -84,7 +76,7 @@ async fn test_with_parquet_word_aligned() -> Result<()> {
 }
 
 async fn get_exec(
-    ctx: &SessionContext,
+    ctx: &SessionState,
```

Comment:
Taking `SessionState` here is more correct as it allows the query to be executed against the same state